sonar-project.properties

```diff
@@ -0,0 +1,9 @@
+sonar.projectKey = yoti-web-sdk:ruby
+sonar.projectName = ruby-sdk
+sonar.projectVersion = 1.5.0
+sonar.language=ruby
+sonar.exclusions = **/protobuf/**.rb, coverage/**, spec/sample-data/**
+sonar.links.scm = https://github.com/getyoti/yoti-ruby-sdk
+sonar.sources = ./lib
+sonar.ruby.coverage.reportPaths = coverage/.resultset.json
+sonar.verbose = true
```

Comment:
Should we also exclude this in the gem spec?

Reply:
> Just the Sonar version
> 
> _(the gemspec exclusions could be done in a follow up ticket - we may want to exclude other files like `.travis.yml`)_

I've added those exclusions.

Reply:
Tested locally and excludes as expected 👍